### PR TITLE
fix some typos, comment mismatch, and small code optimization

### PR DESCRIPTION
**Pull request by javelinjs**
As title

### CODE REVIEW

File: R-package/R/mxnet_generated.R [hunk 1]
```diff
@@ -68,7 +68,7 @@ NULL
 #' @name mx.nd.cos
 NULL
 
-#' Calcuate 2D matrix multiplication
+#' Calculate 2D matrix multiplication
```
Comment (thirdwing) [inline R-package/R/mxnet_generated.R]: This file is generated by `src/ndarray/ndarray.cc` automatically. So if you found another typo in documents, you can just fix the C++ file.

Comment (javelinjs) [inline R-package/R/mxnet_generated.R]: Reverted. Thank you for pointing out.

**Comment by winstywang** [comment]
Please merge latest change, then I can merge

**Comment by javelinjs** [comment]
@winstywang Done. Could you help me to rerun the test? Seems something hanged. Everything passed from my side: https://travis-ci.org/javelinjs/mxnet/builds/104542628

**Pull request merged**